app/Controllers/FrontendController.php

```
@@ -1877,14 +1848,14 @@ public function event(Request $request, Response $response, mysqli $db, array $a
         $seoTitle = $event['seo_title'] ?: ($event['title'] . ' - ' . $appName);
         $seoDescription = $event['seo_description'] ?: $excerpt;
         $seoKeywords = $event['seo_keywords'] ?? '';
-        $seoCanonical = $baseUrl . '/events/' . $event['slug'];
+        $seoCanonical = absoluteUrl(RouteTranslator::route('events') . '/' . $event['slug']);
 
         // Open Graph tags
         $ogTitle = $event['og_title'] ?: $event['title'];
         $ogDescription = $event['og_description'] ?: $seoDescription;
         $ogType = $event['og_type'] ?: 'article';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$ogType'. (undefined)

(UnusedLocalVariable)